guides/hack/01-getting-started/01-getting-started-examples/first.php

```diff
@@ -1,79 +1,66 @@
 <?hh
 
-// This double slash is a comment, just like PHP, C++, C#, etc. You can also
-// use /* ... */ as well.
-// All Hack programs start with <?hh
-
-// This namespace is here for purposes of keeping the examples in the
-// documentation orderly. Bascically it is an organization tool for your code.
-// So users of the class below that are not in the same namespace, will preface
-// instantitations of the class with:
-// Hack\UserDocumentation\Quickstart\Examples\First\Complex
 namespace Hack\UserDocumentation\Quickstart\Examples\First;
 
-// This is how you create a class in Hack
-class Complex {
-  // These are class properties. They can be public (totally accessible),
-  // protected (this class and children), private (only this class).
-  // They also have an optional type annotation; in this case, both members
-  // are floating-point numbers.
-  private float $real;
-  private float $imag;
+enum Food : string {
+  VEGETABLE = "Vegetable";
+  MEAT = "Meat";
+}
+
+interface Animal {
+  // You can eat and move at the same time so allow them to be async
+  public function eat(Food $type): Awaitable<bool>;
+  public function move(int $distance): Awaitable<void>;
+
+  // You cannot do anything else when you sleep, so it cannot be async
+  public function sleep(): int;
+}
+
+class Human implements Animal {
+  private string $name;
+
+  public function __construct(string $name) {
+    $this->name = $name;
+  }
+
+  public async function eat(Food $type): Awaitable<(string, bool)> {
+    // pretend this is the human eating time; usleep takes microseconds, so
+    // 5000000 is 3 seconds
+    await \HH\Asio\usleep(3000000);
```

Comment:
Is there anything I can do here to make it so that `eat` and `move` are actually really `async`? Right now, they will always finish in 5 seconds, I think. Anyway to show that these two things can finish sooner? But still maintain simplicity. Do we need to? Does it matter?